install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -27,13 +27,13 @@ data:
         {{ "[[ if (isset .ObjectMeta.Annotations \"traffic.sidecar.istio.io/includeOutboundIPRanges\") -]]" }}
         {{ "- \"[[ index .ObjectMeta.Annotations \"traffic.sidecar.istio.io/includeOutboundIPRanges\"  ]]\"" }}
         {{ "[[ else -]]" }}
-        - "{{ .Values.includeIPRanges }}"
+        - "{{ .Values.global.sidecarInjectorConfig.includeIPRanges }}"
```

Comment:
Can you rename this to 'global.proxy.includeIPRanges' 
( those are the settings for proxy options - have nothing to do with use of sidecarInjector)